Gem/Nodes/ViewModels/ImportNodeSceneViewModel.swift

```diff
@@ -0,0 +1,139 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import SwiftUI
+import Primitives
+import Components
+import BigInt
+
+class ImportNodeSceneViewModel: ObservableObject {
+    private let nodeService: NodeService
+    private let importNodeService: ImportNodeService
+
+    @Published private var chain: Chain
+
+    @Published var urlString: String = ""
+    @Published var state: StateViewType<Bool> = .noData
+    @Published var chainID: String = ""
```

Comment:
Instead of having chainID, block number and in sync variables, maybe create `ImportNodeResult` that will have those values ?